### DISCUSSION TITLE
openshift-install create install-config not working on 4.13.0-0.okd-2023-09-03-082426

### DISCUSSION THREAD

**Discussion by psgandalf** [okd-project/okd#1743] [Q&A] [answered]
Hi,
When I try to run openshift-install create install-config on my vcenter cluster it finds all vmware datastore things but after entering pull-secret it fails. Full path of datastore must be provided in format "/<datacenter/datastore/<datastore>".
I have an earlier version 4.12.0-0.okd-2023-02-04-212953. With this the config file is created correct.

**Comment by jcpowermac** [comment, marked as answer]
@psgandalf you are hitting this bug https://github.com/openshift/installer/pull/7343 I suspect

you can create the install-config manually to work around the issue.

**Reply by psgandalf** [reply]
Bad research from my siden then. Thx for the bug link. I will follow that issue. I'm all set with the install-config.yaml so this is solved as far as I'm concerned.
